### PR TITLE
[F] Add disabled state for HdLoaderButton

### PR DESCRIPTION
**Pull request by leandroinacio**
https://homeday.atlassian.net/browse/FET-176
-
Not sure why we had a test expecting to trigger the click when the button is disabled, but if there is a reason we should add some comments and a note on storybook.

### CODE REVIEW

**Comment by aym3nb** [comment]
I thought about the same thing when I updated it from myHomeday.
But since the creator left and there are no notes I just figured the desired behavior from the previous usage in myHomeday.
So, I think we need to confirm with designers first.

**Comment by viniciuskneves** [comment]
> @leandroinacio I thought about the same thing when I updated it from myHomeday #374
> But since the creator left and there are no notes I just figured the desired behavior from the previous usage in myHomeday.
> So, I think we need to confirm with designers first.

Check the comments here: https://github.com/homeday-de/homeday-blocks/pull/374#discussion_r400173384

File: src/components/buttons/HdLoaderButton.vue [hunk 1]
```diff
@@ -15,6 +15,7 @@
       @click="clicked"
       @keyup.enter.space="clicked"
       @transitionend="runTransitionQueue"
+      :disabled="disabled"
```
Comment (viniciuskneves) [inline src/components/buttons/HdLoaderButton.vue:18]: If you're just forwarding the prop to the button, I would go to `v-bind="$attrs"` and remove it from the props definition. That is a minor and just a tip, as other button attributes would be automatically forwarded.

**Comment by leandroinacio** [comment]
@aym3nb @viniciuskneves fyi talked to Greg, he said:

> Disabled = grey :heavy_check_mark:
Disabled has no hover state :heavy_check_mark:
Disabled “allows” the click in form of a ripple effect that triggers the form field (the button itself doesn’t change it’s color or type color) + the click requests if anything is properly filled (like on production)

So I'm going to work on these and add a note on storybook.

**Comment by leandroinacio** [comment]
@homeday-de/frontend-engineering ready for another review round :)

File: tests/unit/components/buttons/HdLoaderButton.spec.js [hunk 2]
```diff
@@ -105,4 +105,23 @@ describe('HdLoaderButton', () => {
...
+  it('should toggle disabled class', async () => {
+    // Enabled
+    const disabledClass = 'btn--primary-disabled';
+    const wrapper = wrapperBuilder({
+      props: {
+        disabled: false,
+      },
+    });
+    const button = wrapper.get(BUTTON_SELECTOR);
+    expect(button.attributes().class).not.toContain(disabledClass);
```
Comment (viniciuskneves) [inline tests/unit/components/buttons/HdLoaderButton.spec.js:118]: I think you could have used https://vue-test-utils.vuejs.org/api/wrapper/#classes

Comment (leandroinacio) [reply tests/unit/components/buttons/HdLoaderButton.spec.js]: Yeah makes sense, done 👍

**Pull request merged**